plugins/rapid7_insight_agent/plugin.spec.yaml

```diff
@@ -4,7 +4,8 @@ products: [insightconnect]
 name: rapid7_insight_agent
 title: Rapid7 Insight Agent
 description: Using the Insight Agent plugin from InsightConnect, you can quarantine, unquarantine and monitor potentially malicious IPs, addresses, hostnames, and devices across your organization
-version: 1.0.3
+version: 1.0.4
+supported_versions: ["Rapid7 Insight Agent"]
```

Comment:
Could you please add a date to a `supported_versions`? I mean date of testing the plugin in that format i.e. `2022-04-06`